src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.Searcher.cs

```diff
@@ -64,7 +64,7 @@ private class Searcher
                 }
             }
 
-            internal async void Search()
+            internal async Task SearchAsync()
```

Comment:
out of curiosity... why?  This does seem like an appropriate use of async-void.  No callers use or should use the return value here.  And thsi function takes care of actually ensuring that the async-mechanism it is bridging to is handled.

Reply:
➡️ Two reasons:

1. Previous signature failed to communicate the asynchronous nature of the operation
1. `async void` is forbidden

Reply:
➡️ This is now covered under the VSTHRD002 umbrella. Once enabled, we will have much better analyzer enforcement for the correct implementation of fire-and-forget operations than we do today, and a key part of this enforcement effort is blocking the use of `async void`. The special case is not needed, and works against the goals of consistent, and in this case enforceable, practices.